src/include/processor/operator/persistent/copy_rdf.h

```diff
@@ -1,10 +1,28 @@
 #pragma once
 
+#include "common/enums/scan_source_type.h"
 #include "processor/operator/sink.h"
-
 namespace kuzu {
 namespace processor {
 
+struct CopyRdfPrintInfo final : OPPrintInfo {
+    std::string tableName;
+    common::ScanSourceType copySource;
```

Comment:
We probably don't need to print the source because `Copy` implementation should be independent from `source type`